src/OpenSpaceNet.cpp

```diff
@@ -276,6 +281,7 @@ void OpenSpaceNet::initFeatureSet()
     VectorOpenMode openMode = args_.append ? APPEND : OVERWRITE;
 
     featureSet_ = make_unique<FeatureSet>(args_.outputPath, args_.outputFormat, args_.layerName, definitions, openMode);
+    featureSet_->setProjection(sr_.wkt());
```

Comment:
Also in PR #177, it was something that ended up being necessary anyways. I don't recall why exactly, but the end result was that the underlying OGRSpatialReference can be accessed through a getter in SpatialReference.